mongodb/openapi
# CLOUDP-277336: update endpoint removed change entries to include the sunset date

## Proposed changes



_Jira ticket:_ CLOUDP-277336

This PR proposes the following changes:

- Updates the removed-endpoint text message to include the sunset date.
- Update the `newOasDiffEntriesFromSunsetEndpoints` to add only one changelog entry when the endpoint has reached sunset.


I was able to test the fixes by following these steps:

- download the specs from the changelog GH action
- Update the revision to sunset one endpoint
- Check that the generated changelog has only one entry for the sunset endpoint
